src/Parser/Shaman/Restoration/Modules/Spells/ChainHeal.js

```diff
@@ -39,6 +39,27 @@ class ChainHeal extends Analyzer {
       });
   }
 
+  get suggestionThreshold(){
```

Comment:
I recommend also using these threshold values in the `suggestions()` method so that you have 1 place you need to update should you ever want to change these. Also try to use the `actual` property there if possible.